src/Library/demos/Progress Bar/main.js

```diff
@@ -0,0 +1,34 @@
+const first_bar = workbench.builder.get_object("first");
+const second_bar = workbench.builder.get_object("second");
+const play = workbench.builder.get_object("play");
+
+function handleProgress() {
+  let counter = 0.3;
+  let counter_two = 0.25;
+
+  const intervalId_one = setInterval(() => {
```

Comment:
Add a brief comment above each of these intervals describing what they're doing, to help anyone skimming the source later. For example:

```js
// Advance the first progress bar 10% every 1.5 seconds
```

Reply:
Sure